storybook/test/__snapshots__/index.js.snap

```diff
@@ -1539,61 +1539,78 @@ exports[`Storyshots Components|FontSizePicker Default 1`] = `
 <fieldset
   className="components-font-size-picker"
 >
-  <legend>
+  <legend
```

Comment:
useless snapshots that we change as the implementation is updated?

Reply:
Yes, from the "storyshots".

Reply:
Why useless? I'd expect snapshots to change as the markup changes. I'd also expect to review those changes and make sure they are consistent with what we were trying to do, and nothing changed that shouldn't. That's the whole purpose of snapshots 🙂

Reply:
In my experience, each time you change the implementation, you have to change the snapshot and oftentimes you don't really check the details as it's a big blob of content.

Also, with snapshots it's not clear what are we testing exactly, we're not testing a specific behavior but just random content, it makes it hard to know what's a regression and what's not.

Another question: did you ever found aa regression with these snapshots?

Reply:
I think we use them for things like highlighting missing/removed class names that plugins rely on.

Reply:
My take on it is snapshots are purely for testing markup: making sure no HTML has been accidentally stripped out or made unsemantic, classname or aria attribute changes, etc. It'll only be a big blob if we make a lot of changes! In this instance it's quite big because we changed the markup for the whole component. Still, I find markup changes much more readable in this format than in JSX.

I have found regressions before with snapshots; mostly of the kind mentioned above, but sometimes I've spotted issues with semantics on components being built from scratch too, because the snapshot view is pretty close to the output HTML.

Of course snapshots are only useful when we read them, but that is just a cultural issue that we can work on improving through educating on good testing and code review practice 😀

Reply:
Yeah, this is my take on them too.

Reply:
It's very hard to argue against tests :) and I had this exact same conversation over and over again with different people and everytime we land the snapshots and decide to rollback later :). Nothing personal here but I think this is just one of these occasions.

Reply:
I'd love to know why you decided to roll back! In my previous project snapshots were pretty much universally agreed on. This PR might not be the best place for discussion 😄, but maybe we could bring it up at a Slack meeting at some point?

Reply:
What I said above. basically after some time using them, we noticed that they are not that useful.